NEWS.md

```diff
@@ -1,4 +1,5 @@
 # Version 1.0.9000.9133
+* Modified slot names in sub-classes of `CohortSize` for greater consistency
```

Comment:
that is not the full story and I think we don't need these updates here, as we anyway say that the whole package was refactored. let's rather describe this in the migration vignette

Reply:
Done.